templates/includes/pagination.html

```diff
@@ -0,0 +1,17 @@
+{% if is_paginated %}
+  <ul class="pagination">
+    {% if page_obj.has_previous %}
+      <li class="page-item">
+        <a href="?page= {{ page_obj.previous_page_number }}" class="page-link">prev</a>
```

Comment:
There is an extra space after `?page=` in the `href` attribute. This might cause issues with the URL. Consider removing the space to ensure the URL is correctly formatted.